docs/upload-strategies/tus.mdx

```diff
@@ -0,0 +1,273 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import UppyCdnExample from '/src/components/UppyCdnExample';
+
+# Tus
+
+The `@uppy/tus` plugin brings resumable file uploading with [Tus](http://tus.io) to Uppy by wrapping the [tus-js-client][].
+
+## When should I use it?
+
+:::tip
+Not sure which upload strategy is best for you? Read “[Choosing the upload strategy you need](/docs/guides/choosing-upload-strategy)”.
+:::
+
+[Tus][tus] is an open protocol for resumable uploads built on HTTP.
+This means accidentally closing your tab or losing connection let’s you continue, for instance, your 10GB upload
+instead of starting all over.
+
+It supports any language, any platform, and any network. You can checkout the [implementations](https://tus.io/implementations.html) to
+find the implementation in your favorite language. You can store files on your self-hosted Tus server, but also use service integrations (such as S3) to store files
+externally.
+
+If you want reliable, resumable uploads: use `@uppy/tus` to connect to your Tus server in a few lines of code.
+
+## Install
+
+<Tabs>
+  <TabItem value="npm" label="NPM" default>
+
+  ```shell
+  npm install @uppy/tus
+  ```
+
+  </TabItem>
+
+  <TabItem value="yarn" label="Yarn">
+
+  ```shell
+  yarn add @uppy/tus
+  ```
+
+  </TabItem>
+
+  <TabItem value="cdn" label="CDN">
+    <UppyCdnExample>
+      {`
+        import { Uppy, Tus } from "{{UPPY_JS_URL}}"
+        new Uppy().use(Tus, { endpoint: 'https://tusd.tusdemo.net/files' })
+      `}
+    </UppyCdnExample>
+  </TabItem>
+</Tabs>
+
+## Use
+
+A quick overview of the complete API.
+
+```js {10} showLineNumbers
+import Uppy from '@uppy/core'
+import Dashboard from '@uppy/dashboard'
+import Tus from '@uppy/tus'
+
+import '@uppy/core/dist/style.min.css'
+import '@uppy/dashboard/dist/style.min.css'
+
+const uppy = new Uppy
+  .use(Dashboard, { inline: true, target: 'body' })
+  .use(Tus, { endpoint: 'https://tusd.tusdemo.net/files/' })
+```
+
+## API
+
+### Options
+
+:::info
+All options are passed to `tus-js-client` and we document the ones here that we added or changed.
```

Comment:
If you say that you only mention options differing from tus-js-client, then why does it mention endpoint,, overridePatchMethod, and retryDelays?

Reply:
Good point! I should reword that we also document required options, like endpoint. The `retryDelays` default is different than the tus-js-client default. `overridePatchMethod` can indeed be removed.